src/core/Akka/Configuration/Config.cs

```diff
@@ -483,6 +483,11 @@ public virtual bool HasPath(string path)
                 current = current.Fallback;
             }
         }
+
+        /// <summary>
+        /// A static "Empty" configuration we can use instead of <c>null</c> in some key areas.
+        /// </summary>
+        public static readonly Config Empty = ConfigurationFactory.Empty;
```

Comment:
Technically we don't need this - I can just use `ConfigurationFactory.Empty` instead. I added this so long ago that it's tough for me to remember why I added it :\